packages/patternfly-4/react-charts/src/components/ChartUtils/chart-label.ts

```diff
@@ -1,23 +1,24 @@
+import { ChartLegendPosition } from "../ChartLegend";
 import { ChartCommonStyles } from '../ChartTheme';
-import { ChartThemeDefinition } from '../ChartTheme';
 
-interface ChartLabelDimensionsInterface {
-  label: string; // The label text
-  theme: ChartThemeDefinition; // The theme that will be applied to the chart
-}
+export enum ChartLabelPosition {
+  bottom = 'bottom',
+  center = 'center',
+  right = 'right'
+};
 
 interface ChartLabelPaddingXInterface {
   dx?: number; // Horizontal shift from the x coordinate
   chartWidth: number; // Width of chart (e.g., donut) within SVG
-  labelPosition: string; // Position of label (e.g., bottom, right)
-  legendPosition: string; // Position of legend (e.g., bottom, right)
+  labelPosition?: string; // Position of label (e.g., bottom, right)
```

Comment:
can we type the possible positions out for the positional type props? e.g. 'bottom' | 'right'| etc

Reply:
I agree this should be fixed sooner rather than later. We can make a `type Position = 'bottom' | 'right' | 'left' | 'top';`.

Reply:
This is just a utility function. The ChartDonut's `legendPosition` and `subTitlePosition` props are already typed with the component.

subTitlePosition?: 'bottom' | 'center' | 'right';
legendPosition?: 'bottom' | 'right';

If I want to continue using the enum below, the utility must use a `string` type.

```
export enum ChartDonutSubTitlePosition {
  bottom = 'bottom',
  center = 'center',
  right = 'right'
}
```

Otherwise, I cannot refer to `ChartLabelPosition.center` in my switch statements. To make that work, I also need to change the code to compare strings (e.g., `'center'`) instead.

That said, the donut component supports multiple legend positions. It's possible that the legendPosition value could be 'bottom', 'bottom-left', 'right', etc. However, there is no where else in `chart-label` utils that we refer to 'bottom-left' or 'right'.

I don't want to maintain unused legend position types here.

The only special legend position for chart-label utils is `bottom`. All other positions use the default position; thus, the `string` type seems appropriate here.

Reply:
I've expanded the utility function types as `'bottom' | 'right'` etc., but reverted the switch statements to compare raw strings.